### PR TITLE
Feature/get a type by type ngsiv2

### PR DESCRIPTION
**Pull request by iariasleon**
**WARNING: verify that the PR https://github.com/telefonicaid/fiware-orion/pull/2049 is merged before this**

```
                     SUMMARY:
-------------------------------------------------
  - components\ngsiv2\types\retrieve_an_entity_type\retrieve_an_entity_type.feature >> passed: 64, failed: 0, skipped: 1 and total: 65 with duration: 23.611 seconds.
  - components\ngsiv2\types\retrieve_an_entity_type\retrieve_an_entity_type_traces_in_log.feature >> passed: 1, failed: 0, skipped: 0 and total: 1 with duration: 6.802 seconds.
-------------------------------------------------
  Date: Wednesday, April 20, 2016 17:44:47
-------------------------------------------------
2 features passed, 0 failed, 0 skipped
65 scenarios passed, 0 failed, 1 skipped
627 steps passed, 0 failed, 9 skipped, 0 undefined
Took 0m30.413s
```
@crbrox @kzangeli

### CODE REVIEW

**Comment by jcanonav** [comment]
LGTM

**Comment by kzangeli** [comment]
2049 is merged.   Refresh needed

File: test/acceptance/behave/components/common_steps/entities/get_steps.py [hunk 1]
```diff
@@ -209,8 +223,8 @@ def verify_that_the_attribute_value_by_id_is_returned(context):
     __logger__.info("...Verified the attribute value by ID returned from a request...")
 
 
-@step(u'verify that entity types are returned in response are: "([^"]*)"')
-def verify_that_attribute_types_are_returned_in_response(context, types):
+@step(u'verify that entity types returned in response are: "([^"]*)"')
+def verify_thate_ntitity_types_are_returned_in_response(context, types):
```
Comment (kzangeli) [inline test/acceptance/behave/components/common_steps/entities/get_steps.py]: verify_thate_ntitity_types_are_returned_in_response  => verify_that_entity_types_are_returned_in_response   ?

Comment (iariasleon) [inline test/acceptance/behave/components/common_steps/entities/get_steps.py]: Fixe in aa22b63

**Pull request merged**